orchagent/portsorch.cpp

```diff
@@ -3130,6 +3139,36 @@ void PortsOrch::doTask(Consumer &consumer)
     }
 }
 
+bool PortsOrch::flushFdbEntries(Port port)
+{
+    vector<sai_attribute_t>    attrs;
+    sai_attribute_t            attr;
+    sai_status_t               rv = SAI_STATUS_SUCCESS;
+
+    SWSS_LOG_ENTER();
+
+    if (SAI_NULL_OBJECT_ID == port.m_bridge_port_id)
+    {
+        SWSS_LOG_WARN("Couldn't flush FDB entries for port: %s", port.m_alias.c_str());
+        return false;
```

Comment:
IMO, this should return 'true'. We dont want retries etc to happen if there is no bridge_port